themes/base.theme.bash

```diff
@@ -98,6 +98,44 @@ function scm_prompt_info {
   [[ $SCM == $SCM_SVN ]] && svn_prompt_info && return
 }
 
+function scm_prompt_status {
+  scm
+  scm_prompt_char
+  SCM_DIRTY=0
+  SCM_STATE=''
+  [[ $SCM == $SCM_GIT ]] && git_prompt_status && return
+  [[ $SCM == $SCM_HG ]] && hg_prompt_info && return
```

Comment:
Can you please add a `TODO` comment above this line to show that you're still using the full version for `hg` here?